css/main.css

```diff
@@ -3,18 +3,26 @@
 	--base-color: #E5D9B9;
 	--lighter-color: #FBEFD5;
 	--lightest-color: #FFF5B5;
-	--topbar-height: 2.5em;
 }
 
 html{
 	font-family: Calibri,serif;
-	scroll-padding-top: var(--topbar-height);/*below topbar*/
+	width: 100vw;
+	height: 100vh;
+	overflow: hidden;
 }
 
 body{
 	background-color: var(--base-color);
-	margin-top: 0;
+	margin: 0;
```

Comment:
can we do "margin: 0 0.5em" ? or are we avoiding short hands?

Reply:
in another comment I mentioned more in depth my thoughts on this. We can ask the rest of the team, for now I'll leave the shorthand suggestions unresolved so I can update them depending on what we all decide. It's a good conversation to figure out.

Reply:
wait that works?
oh huh. 
yeah we can do that

Reply:
Sounds like from a quick poll from the dev team that shorthand in css is preferred so I'm adding this change in.